src/provider_references.jl

```diff
@@ -0,0 +1,74 @@
+import Tokenize.Tokens
```

Comment:
it's valid for now (might not always be if the way module imports work w.r.t. one another changes) but a little odd, and would break if there were ever a future version of CSTParser that did not itself depend on Tokenizer